### PR TITLE
Add Validator and Nominator guide

### LINKED ISSUE #282: Write a doc about testing staking/bonding/nominating mechanism
Need to write a step-by-step guide to verify staking mechanism is working well.

In the second stage, we need to describe all procedures within the staking workflow.

Use https://hackmd.io for the draft.

### PR DESCRIPTION
**Pull request by EugenWay**
Resolves #282.

### CODE REVIEW

**Review by AndrePanin — CHANGES_REQUESTED** [review]
Pls add a few sentences about who are nominators, why are they needed on the network, what does it give to the nominator himself, for what and how does he receive awards. 
The same about validators - why are they needed, what functions perform, what they will get for validation etc.

**Comment by NikVolf** [comment]
Please move this also to Vara-related resources

**Review by AndrePanin — COMMENTED** [review]
added some text edits. 
need to add Get Payouts part, @EugenWay

**Review by AndrePanin — APPROVED** [review]

**Review by NikVolf — CHANGES_REQUESTED** [review]
should not be on gear-tech

File: docs/gear-networks/VARA/validator.mdx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+```
+
+</TabItem>
+
+<TabItem value="mac-linux" label="Linux x64">
+
+Terminal:
+
+```bash
+wget https://github.com/gear-tech/builds/releases/download/vara/vara-nightly-linux-x86_64.tar.xz && \
```
Comment (btwiuse) [inline docs/gear-networks/VARA/validator.mdx:105]: download link is broken, maybe switch to https://get.gear.rs?

**Comment by EugenWay** [comment]
All docs related to Vara Network was moved to https://github.com/gear-tech/vara-wiki

**Pull request closed without merging**